manifests/0000_50_olm_07-collect-profiles.cronjob.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: collect-profiles
+  namespace: openshift-operator-lifecycle-manager
+spec:
+  schedule: "*/15 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: collect-profiles
+          containers:
+            - name: collect-profiles
```

Comment:
This pod template needs one of the allowed priority classes:
```
4 pods found with invalid priority class (should be openshift-user-critical or begin with system-): openshift-operator-lifecycle-manager/collect-profiles-27107640-b6f84 (currently "") openshift-operator-lifecycle-manager/collect-profiles-27107640-ls2rz (currently "") openshift-operator-lifecycle-manager/collect-profiles-27107655-sg82n (currently "") openshift-operator-lifecycle-manager/collect-profiles-27107655-tdmzg (currently "")
```
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_operator-framework-olm/112/pull-ci-openshift-operator-framework-olm-master-e2e-gcp/1416088009018707968#1:build-log.txt%3A5297

Reply:
I have set the priority class to `openshift-user-critical`, which will allow the job to be preempted by user workloads and OOMKilled.